src/ECharts/Theme.purs

```diff
@@ -0,0 +1,33 @@
+module ECharts.Theme
+  ( Theme(..)
+  , dark
+  , infographic
+  , macarons
+  , roma
+  , shine
+  , vintage
+  ) where
+
+import Data.Foreign (Foreign)
+
+foreign import forceExport :: Foreign
```

Comment:
did this to make sure theme.js was included in output. is non hacky  way to do it?